usr/lib/cca_stdll/cca_specific.c

```diff
@@ -4511,11 +4512,15 @@ static CK_RV cca_get_acp_info_handler(STDLL_TokData_t *tokdata,
         role_info = (struct cca_role_data *)role_data;
 
         /* Identify default role */
-        if ((memcmp(role_name, "DFLT", 4) != 0 &&    /* since z13 */
+        if ((memcmp(role_name, "DFLT", 4) != 0 &&    /* since z13 + non-s390x */
+#if defined(__s390__)
              memcmp(role_name, "DEFALT", 6) != 0) || /* before z13 */
-            role_info->lower_time_limit != 0x0000 ||
-            (role_info->upper_time_limit != 0x0000 &&
-             role_info->upper_time_limit != 0x173b) || /* 23:59 */
+#else
+             memcmp(role_name, "DEFAULT", 7) != 0) || /* non-s390x */
```

Comment:
Just check: in line 4515/4517 the keyword is "DEFALT", here it is "DEFAULT".

Reply:
Yes thats the reason for the #if statement :-) 
For s390x its 'DEFALTxx' where xx is the domain, while for non#s390 its just 'DEFAULT ' (no domain).